_search-plugins/sql/cli.md

```diff
@@ -71,32 +62,46 @@ For a list of all available configurations, see [clirc](https://github.com/opens
 
 ## Using the CLI
 
-1. Save the sample [accounts test data](https://github.com/opensearch-project/sql/blob/main/doctest/test_data/accounts.json) file.
-
-1. Index the sample data.
+1. Run the CLI tool. If you cluster runs with default security settings, use following command line:
```

Comment:
"If your cluster runs with the default security settings, use the following command:"

Reply:
Thank you, fixed in d5e9d0d.